app/src/main/java/com/example/hw_3/fragment/FragmentList.kt

```diff
@@ -0,0 +1,136 @@
+package com.example.hw_3.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.SearchView
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hw_3.paging.PagingData
+import com.example.hw_3.person.PersonGitHub
+import com.example.hw_3.R
+import com.example.hw_3.retrofit.GitService
+import com.example.hw_3.adapter.PersonAdapter
+import com.example.hw_3.databinding.FragmentListBinding
+import com.google.android.material.snackbar.Snackbar
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class FragmentList : Fragment() {
+    private var _binding: FragmentListBinding? = null
+
+    private val binding get() = requireNotNull(_binding) {
+        "View does not exist anymore"
+    }
+
+    private val adapter = PersonAdapter { person ->
+        findNavController().navigate(
+            R.id.details, bundleOf("name" to person.login)
+        )
+    }
+
+    private var isLoading = false
+    private var currentPage = 0
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentListBinding.inflate(inflater, container, false)
+            .also { binding ->
+                _binding = binding
+            }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        loadPersons()
+
+        with(binding) {
+
+            val linearLayoutManager = LinearLayoutManager(
+                view.context, LinearLayoutManager.VERTICAL, false
+            )
```

Comment:
используется стандартный конструктор, можно проще
```suggestion
            val linearLayoutManager = LinearLayoutManager(view.context)
```

Reply:
не исправлено, можно вообще через xml задавать